src/nrniv/bgpdma.cpp

```diff
@@ -533,39 +422,21 @@ double nrn_bgp_receive_time(int type) { // and others
 		break;
 #endif
 	case 8: // exchange method properties
-		// bit 0-1: 0 allgather, 1 multisend as MPI_ISend,
-		// 2 multisend DCMF, 3 multisend DCMF with record replay
+		// bit 0: 0 allgather, 1 multisend (MPI_ISend)
+		// bit 1: unused, legacy
 		// bit 2: n_bgp_interval, 0 means one interval, 1 means 2
 		// bit 3: number of phases, 0 means 1 phase, 1 means 2
 		// bit 4: 1 means althash used
 		// bit 5: 1 means enqueue separated into two parts for timeing
 	    {
-		int meth = use_dcmf_record_replay ? 3 : use_bgpdma_;
+		int meth = use_bgpdma_ ? 1 : 0;
 		int p = meth + 4*(n_bgp_interval == 2 ? 1 : 0)
```

Comment:
nitpick
```suggestion
		int p = int(use_bgpdma_) + 4*(n_bgp_interval == 2 ? 1 : 0)
```

Reply:
is it really more clear like that?

Reply:
Or, at least I would rename `meth` to `method`